app/controllers/placements_controller.rb

```diff
@@ -5,23 +5,63 @@ class PlacementsController < ApplicationController
 
   rescue_from ActiveRecord::RecordNotFound, with: -> { head :not_found }
 
+  def index
+    authorize @casa_case
+    @placements = @casa_case.placements.includes(:placement_type).order(placement_started_at: :desc)
```

Comment:
Typically, no `authorize` call in index action. Better to just policy_scope the collection of records.  You will need to implement a PlacementPolicy::Scope, and then:

```suggestion
    @placements = policy_scope(@casa_case.placements).includes(:placement_type).order(placement_started_at: :desc)
```

Reply:
You could do both, like if limiting the view to admin/supervisor, with `authorize :placement` or `authorize Placement`